programming_guide/section-4/section-4b/Makefile

```diff
@@ -59,7 +59,7 @@ trace: ${targetname}.exe build/trace.xclbin
 	${powershell} ./$< -x build/trace.xclbin -i build/insts.txt -k MLIR_AIE -t ${trace_size}
 	../../../programming_examples/utils/parse_trace.py --filename trace.txt --mlir build/aie_trace.mlir --colshift 1 > trace_4b.json
 
-trace_py: build/final.xclbin
+trace_py: build/trace.xclbin
```

Comment:
Looks like I missed this in #1651 :(

Reply:
On a side note, I'd like to add more trace testing to make sure changes in the trace.py and parse_trace.py don't break things. I changed the order of events so that order listed in the events array now match the order on the output (instead of matching the config register order which is reversed).